spec/requests/categories/create_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe 'Category create', type: :request do
+  let(:category) { create(:category) }
+
+  context 'with valid params' do
+    let(:params) do
+      { category: { name: :name } }
+    end
+
+    it 'creates category ' do
+      expect do
+        post '/categories', params: params
+      end.to change(Category, :count).by(1)
+
+      expect(response).to redirect_to(categories_path)
+      follow_redirect!
+
+      category = Category.last.name
+      expect(category).to be_a(String)
+      expect(response.body).to include(category)
+      expect(response.body).to include('Категория &quot;name&quot; добавлена')
```

Comment:
Rails already checks that you have string value for `name`. It will help you to check that you've created category with correct name. Also it's better to load category and the check `expect(response.body).to include(category.name)` to have possibility to extend this text with more expectations.

```suggestion
      category = Category.last

      expect(category).to have_attributes(params[:category])

      expect(response.body).to include(category.name)
      expect(response.body).to include("Категория &quot;#{category.name}&quot; добавлена")
```